Sources/ComposableArchitecture/SwiftUI/IdentifiedArray.swift

```diff
@@ -237,6 +237,20 @@ where Element: Identifiable, ID == Element.ID {
   public init() {
     self.init([], id: \.id)
   }
+
+  public mutating func replaceSubrange<C, R>(_ subrange: R, with newElements: C) where C : Collection, R : RangeExpression, Self.Element == C.Element, Self.Index == R.Bound {
+    let replacingIds = self.ids[subrange]
+    let newIds = newElements.map(\.id)
```

Comment:
We'd like to support Xcode 11.3 for a bit, which contains Swift 5.1 and doesn't have this nice key path functionality. Let's use the old style instead:
```suggestion
    let newIds = newElements.map { $0.id }
```